File: backend/BowlingBackend/Data/Bowler.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Runtime.CompilerServices;

namespace BowlingBackend.Data
{
    public class Bowler
    {
        [Key]
        [Required]
        public int BowlerID { get; set; }
        public string? BowlerLastName { get; set; }
        public string? BowlerFirstName { get; set; }
        public string? BowlerMiddleInit {  get; set; }
        public string? BowlerAddress { get; set; }
        public string? BowlerCity { get; set; }
        public string? BowlerState { get; set; }
        public string? Bowlerzip {  get; set; }
        public string? BowlerPhoneNumber { get; set; }

        [ForeignKey("TeamID")]
        public int? TeamID { get; set; }
        public Team? Teams { get; set; }
    }
}

File: frontend/src/types/Bowler.ts
export type Bowler = {
  bowlerID: number;
  bowlerLastName: string;
  bowlerFirstName: string;
  bowlerMiddleInit: string;
  bowlerAddress: string;
  bowlerCity: string;
  bowlerState: string;
  bowlerzip: string;
  bowlerPhoneNumber: string;
  teamID: number;
  teams: {
    teamdId: number;
    teamName: string;
  };
};
